metadata_mapper/mappers/mapper.py

```diff
@@ -66,6 +66,8 @@ class Record(object):
     def __init__(self, col_id, record):
         self.collection_id = col_id
         self.source_metadata = record
+        self.pre_mapped_data = {}
+        self.enrichment_report = []
```

Comment:
This is that enrichment exceptions error reporting thing. 

pre_mapped_data exists for the sake of some enrichments that happen prior to mapping. it's a stop gap. we should get rid of it.

Reply:
got it